bindings/python/tests/test_pydantic.py

```diff
@@ -0,0 +1,77 @@
+from datetime import datetime
+from typing import List, Optional
+
+import dlite
+from dlite.utils import pydantic_to_metadata, pydantic_to_instance
+
+# Skip test if pydantic isn't installed
+try:
+    from pydantic import BaseModel, Field
+except ImportError:
+    import sys
+    sys.exit(44)
+
+
+class TransformationStatus(BaseModel):
```

Comment:
This is not used in the test and can be removed

Reply:
The `TransformationStatus` class is used to instantiate `t`. I improved the test with additional assert statements.